src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -113,34 +193,64 @@ Translation<Dim>::jacobian(
     const std::unordered_map<
         std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
         functions_of_time) const {
-  if (f_of_r_ == nullptr) {
-    return identity<Dim>(dereference_wrapper(source_coords[0]));
-  } else {
-    std::array<tt::remove_cvref_wrap_t<T>, Dim> distance_to_center{};
-    for (size_t i = 0; i < Dim; i++) {
-      gsl::at(distance_to_center, i) =
-          gsl::at(source_coords, i) - gsl::at(center_, i);
-    }
-    auto radius = magnitude(distance_to_center);
+  // If an inner radius is not specified then take the jacobian of the
+  // MathFunction specific translation.
+  if (!inner_radius_.has_value()) {
+    if (f_of_r_ == nullptr) {
+      return identity<Dim>(dereference_wrapper(source_coords[0]));
+    } else {
+      std::array<tt::remove_cvref_wrap_t<T>, Dim> distance_to_center{};
+      for (size_t i = 0; i < Dim; i++) {
+        gsl::at(distance_to_center, i) =
+            gsl::at(source_coords, i) - gsl::at(center_, i);
+      }
+      auto radius = magnitude(distance_to_center);
+      const DataVector function_of_time =
+          functions_of_time.at(f_of_t_name_)->func(time)[0];
 
+      auto result = make_with_value<
+          tnsr::Ij<tt::remove_cvref_wrap_t<T>, Dim, Frame::NoFrame>>(
+          dereference_wrapper(source_coords[0]), 0.0);
+      for (size_t i = 0; i < Dim; i++) {
+        for (size_t j = 0; j < Dim; j++) {
+          for (size_t k = 0; k < get_size(radius); k++) {
+            if (get_element(radius, k) > 1.e-13) {
+              result.get(i, j) =
```

Comment:
Write a test that would have caught the missing `get_element(..., k)` here and in various similar places.

Reply:
I think the function `test_coordinate_map_argument_types` catches the missing `get_element(..., k)` and I use it for all the different maps.

Reply:
Except it apparently doesn't, because the test is passing?

Reply:
hmmm, I don't understand what you're asking then

Reply:
The LHS of this assignment should be wrapped in `get_element`, but the tests are passing even though it isn't.

Reply:
oh okay, that makes more sense